endpointlogger/logger.go

```diff
@@ -0,0 +1,43 @@
+package endpointlogger
```

Comment:
Can you think of moving this logic to the webrpc generated code?

What can we do to get the context values set by the webrpc handlers from an outside middleware? If we could solve this.. I think it would be very useful for ACL/telemetry/logging too.

Reply:
OK cool -- so can you move this to marketplace-api for now?

Reply:
OK, I added a notice and let's merge for now.

https://github.com/0xsequence/go-libs/pull/14/commits/f2e292f8c76060f2c218cdd6c6e7e9a40608de35